opg-golang-alpine/Makefile

```diff
@@ -1,8 +1,20 @@
 .PHONY: build
+SHELL = '/bin/bash'
 
+mkfile_path := $(abspath $(lastword $(MAKEFILE_LIST)))
+current_dir := $(notdir $(patsubst %/,%,$(dir $(mkfile_path))))
+image := $(current_dir)
 registryUrl ?= registry.service.opg.digital
-image = opg-golang-alpine
+SEED := $(shell /bin/bash -c "echo $$RANDOM")
 
 build:
-	docker build --pull $(no-cache) -t "$(registryUrl)/$(image)" .
-	# docker tag "$(registryUrl)/$(image)" "$(registryUrl)/$(image):$(newtag)"
+	docker-compose -f tests/docker-compose.yml build
+
+test:
+	echo "Seed: ${SEED}"
+	docker-compose -p "${SEED}" -f tests/docker-compose.yml up -d
+	inspec exec tests -t docker://${SEED}_basegen_1
```

Comment:
best not to guess container name and use 
```
docker-compose ps | grep basegen | awk '{ print $1 }'
```

Reply:
We're not actually guessing the container name here. It's based on the seed.

If we used the grep|awk method we could possibly stop other tests suites in the middle of running.